workers/data_refinery_workers/processors/utils.py

```diff
@@ -263,8 +263,8 @@ def end_job(job_context: Dict, abort=False):
         success = True
 
     if not abort:
-        if job_context.get("success", False) and not (job_context["job"].pipeline_applied in ["SMASHER", "QN_REFERENCE", "COMPENDIA"]):
-            
+        if job_context.get("success", False) and not (job_context["job"].pipeline_applied in ["SMASHER", "QN_REFERENCE", "COMPENDIA", "JANITOR"]):
```

Comment:
Also wasn't sure how this specifically related to the ENA RNA-seq fallback

Reply:
It's not but I didn't wanna make a whole other branch for just this change.